sdk/storage/azure-storage-blobs/test/ut/block_blob_client_test.cpp

```diff
@@ -1509,6 +1509,22 @@ namespace Azure { namespace Storage { namespace Test {
         blobItem.Details.RehydratePriority.Value(), Blobs::Models::RehydratePriority::Standard);
   }
 
+  TEST_F(BlockBlobClientTest, DISABLED_SetTierCold)
```

Comment:
Make sure you've tracked all disabled test cases https://github.com/Azure/azure-sdk-for-cpp/issues/3592

Reply:
Yeah, I have tracked